docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -89,27 +91,27 @@ we need to create an `.env` file in the same place as the ipynb notebook.
 
 Save your Twitter API Bearer token in the `.env` file:
 
-```
+```bash linenums="1" hl_lines="4-4" title="someting.env"
 TWITTER_API_BEARER_TOKEN="XXXXX"
 ```
 
 In the notebook, we need to first load the dotenv extension. 
 
-```python
-%load_ext dotenv
-%dotenv
+```bash linenums="1" hl_lines="4-4" title="python"
+load_ext dotenv
+dotenv
 ```
 
 Then, we can get the value of the `TWITTER_API_BEARER_TOKEN` with:
 
-```python
+```bash linenums="1" hl_lines="4-4" title="python"
```

Comment:
and here

Reply:
I think you're looking at outdated file but I changed all python to py